Modules/TOF/src/TaskRaw.cxx

```diff
@@ -47,27 +47,16 @@ void RawDataDecoder::decode()
 
 void RawDataDecoder::headerHandler(const CrateHeader_t* crateHeader, const CrateOrbit_t* crateOrbit)
 {
-  // DRM Counter
-  if (crateHeader->slotPartMask & 0 << 0) {
-    mDRMCounter[crateHeader->drmID].Count(0);
-  }
-
-  // LTM Counter
-  if (crateHeader->slotPartMask & 1 << 0) {
-    mLTMCounter[crateHeader->drmID].Count(0);
-  }
-
-  // TRM Counter
-  for (int i = 1; i < 11; i++) {
-    if (crateHeader->slotPartMask & 1 << i) {
-      mTRMCounter[crateHeader->drmID][i - 1].Count(0);
-    }
-  }
-
-  // Paricipating slot
-  for (int ibit = 0; ibit < 11; ++ibit) {
+  //Participating slot
+  for (int ibit = 0; ibit < 11; ++ibit) {    
```

Comment:
Why not using the loop for TRM only and the non loop for the LTM so as to get rid of the `if(ibit == 0)`?

Reply:
Ok makes sense will fix this, it makes a difference because this way it is slowed down by the fact that every time it has to check if the bit is 0 right?

Reply:
I did this because I tought it would be nice to put toghether the loop for TRM counter and SlotPartMask, but I can leave them togheter and put LTM outside the loop

Reply:
exactly, I would leave the loop for TRMs only, it's also more clear in the code that you are looking at TRMs since you are looping on them